run/README.md

```diff
@@ -0,0 +1,131 @@
+<img src="https://avatars2.githubusercontent.com/u/2810941?v=3&s=96" alt="Google Cloud Platform logo" title="Google Cloud Platform" align="right" height="96" width="96"/>
+
+# Google Cloud Run Node.js Samples
+
+[Cloud Run][run_docs] stateless containers on a fully managed environment or in your own GKE cluster.
```

Comment:
Nit: Cloud Run _runs_?

Also: worth linking to Docker here, and/or an explanation of what containers are? (e.g. [this page](https://cloud.google.com/containers/))

Reply:
Whoa, I didn't know about that page! Thanks.